src/apis/org_service_api.rs

```diff
@@ -184,6 +200,16 @@ pub enum UpdateOrgError {
     UnknownValue(serde_json::Value),
 }
 
+/// struct for typed errors of method [`update_org`]
+#[derive(Debug, Clone, Serialize, Deserialize)]
+#[serde(untagged)]
+pub enum SubscribeOrgToRoleMappingError {
+    Status400(crate::models::BadUpdateOrgRequest),
```

Comment:
Should this be a `BadUpdateOrgRequest` error here?

Reply:
hits the same route, to I used the same error type